mkdocs/docs/HPC/index.md

```diff
@@ -1,10 +1,32 @@
-# Welcome
+---
+hide: "toc"
+---
+# HPC Documentation
+
+---
+
+
+![image](img/ch0-hpc.png){: align=left style="height:383px;width:378px;border-radius:6px" }
+<!-- 1150 1134 -->
 
 Welcome to the {{ hpcinfra }} documentation!
 
 Use the menu on the left to navigate, or use the search box on the top right.
 
-The current view of the documentation is intended for people using **{{ OS }}** as operating system.
-To switch to different operating system (Windows, macOS, Linux), click [here](../#force_select_OS).
+You are viewing documentation intended for people using **{{ OS }}**. To switch to a different operating system (Windows, macOS, Linux), use the OS dropdown in the top bar.
```

Comment:
don't list options here, results in 3 lines rather than 2:

```suggestion
You are viewing documentation intended for people using **{{ OS }}**.
Use the OS dropdown in the top bar to switch to a different operating system.